docs/reference/query-languages/esql/kibana/definition/functions/multi_match.json

```diff
@@ -23,7 +23,7 @@
           "type" : "function_named_parameters",
           "mapParams" : "{name='fuzziness', values=[AUTO, 1, 2], description='Maximum edit distance allowed for matching.'}, {name='auto_generate_synonyms_phrase_query', values=[true, false], description='If true, match phrase queries are automatically created for multi-term synonyms. Defaults to true.'}, {name='minimum_should_match', values=[2], description='Minimum number of clauses that must match for a document to be returned.'}, {name='fuzzy_transpositions', values=[true, false], description='If true, edits for fuzzy matching include transpositions of two adjacent characters (ab → ba). Defaults to true.'}, {name='tie_breaker', values=[0], description='Controls how score is blended together between field groups. Defaults to 0 (best score from each group).'}, {name='type', values=['best_fields'], description='Controls the way multi_match is executed internally. Can be one of `best_fields`, `most_fields`, `cross_fields`, `phrase`, `phrase_prefix` or `bool_prefix`. Defaults to 'best_fields'. See <<multi-match-types,multi_match types>>.'}, {name='lenient', values=[true, false], description='If false, format-based errors, such as providing a text query value for a numeric field, are returned. Defaults to true.'}, {name='operator', values=[AND, OR], description='Boolean logic used to interpret text in the query value. Defaults to OR.'}, {name='max_expansions', values=[50], description='Maximum number of terms to which the query will expand. Defaults to 50.'}, {name='analyzer', values=[standard], description='Analyzer used to convert the text in the query value into token. Defaults to the index-time analyzer mapped for the field. If no analyzer is mapped, the index’s default analyzer is used.'}, {name='boost', values=[2.5], description='Floating point number used to decrease or increase the relevance scores of the query.'}, {name='fuzzy_rewrite', values=[constant_score_blended, constant_score, constant_score_boolean, top_terms_blended_freqs_N, top_terms_boost_N, top_terms_N], description='Method used to rewrite the query. See the rewrite parameter for valid values and more information. If the fuzziness parameter is not 0, the match query uses a fuzzy_rewrite method of top_terms_blended_freqs_${max_expansions} by default.'}, {name='prefix_length', values=[1], description='Number of beginning characters left unchanged for fuzzy matching. Defaults to 0.'}",
           "optional" : true,
-          "description" : "(Optional) Additional options for MultiMatch, passed as <<esql-function-named-params,function named parameters>>.\"  See <<query-dsl-multi-match-query,multi-match query>> for more information."
+          "description" : "(Optional) Additional options for MultiMatch, passed as <<esql-function-named-params,function named parameters>>.{applies_to}`stack: coming`\"  See <<query-dsl-multi-match-query,multi-match query>> for more information."
```

Comment:
@drewdaemon @stratoula FYI we'll start having more of these little applies_to syntaxes in the ESQL docs, but y'all will need to filter them out for Kibana 

This example uses [inline](https://elastic.github.io/docs-builder/syntax/applies/#inline-applies-to) annotation: 

```markdown
{applies_to}`foo: bar`
```

There are also [section](https://elastic.github.io/docs-builder/syntax/applies/#sections) annotations: 

``````markdown
```{applies_to}
stack: ga 9.1
```
``````

Reply:
@craigtaverner can you filter them out for us? Or replace with the correct value?

Reply:
yes if ES can help as they do with other similar things would be highly appreciated

Reply:
@stratoula I think it's now stripping the applies_to metadata correctly from the Kibana files, can you have a look please?

Sorry about the noise from the "Do not edit this" change :/